modules/init.d.js

```diff
@@ -0,0 +1,16 @@
+function init_d() {

```

Comment:
There are two init_d functions in this pull request. This may lead to side effects, best to name them differently if they are to be called separately, or combine them into one if they are to be called together.